src/Webform.js

```diff
@@ -811,6 +811,18 @@ export default class Webform extends NestedDataComponent {
     this.setSubmission(submission);
   }
 
+    /**
+   * @param submission
+   * @param flags
+   * @return {void}
```

Comment:
this is a HUGE nit, but the incoming PR from Travis will cause the linter to complain about this - can we add descriptions for each?

Reply:
Done!